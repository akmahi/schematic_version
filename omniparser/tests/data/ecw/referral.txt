apuid|RecordID|ReferralID|patientID|PatientAccountNo|SpecialityCode|Speciality|ReferralType|ReferralStatus|ReferralFromName|ReferralFromNPI|ReferralToName|ReferralTONPI|FacilityFrom|FacilityTo|ReferralDate
305922|305922.533210.54413|54413|533210|533210|28|Gastroenterology|Outgoing|Pending|Fraser-Farmer, Gail|1841223997|||26|0|2022-10-17
305922|305922.552089.54414|54414|552089|552089|77|Optometrist|Outgoing|open|Brown, Matthew|1396385019|||12|0|2022-10-17
305922|305922.552089.54415|54415|552089|552089|115|Urology|Outgoing|open|Brown, Matthew|1396385019|||12|0|2022-10-17
305922|305922.547456.54417|54417|547456|547456|93|Physical Therapist|Outgoing|open|Pomerantz, Janet|1922181510|||25|0|2022-10-17
305922|305922.552353.54418|54418|552353|552353|162|Speech Therapy|Outgoing|open|Pomerantz, Janet|1922181510|||25|0|2022-10-17
305922|305922.550284.54419|54419|550284|550284|77|Optometrist|Outgoing|open|Brown,Matthew|1396385019|||12|0|2022-10-17
305922|305922.533974.54420|54420|533974|533974|135|Surgery|Outgoing|open|Fraser-Farmer, Gail|1841223997|||26|0|2022-10-17
305922|305922.533974.54421|54421|533974|533974|160|Podiatry|Outgoing|Pending|Fraser-Farmer, Gail|1841223997|||26|0|2022-10-17
305922|305922.14955.54422|54422|14955|524156|19|Dermatology|Outgoing|open|Fraser-Farmer, Gail|1841223997|||26|0|2022-10-17
305922|305922.536047.54423|54423|536047|536047|25|Endocrinology|Outgoing|open|Fortin,Nancy|1679623896|||10|27|2022-10-17
305922|305922.536403.54424|54424|536403|536403|77|Optometrist|Outgoing|open|Brown,Matthew|1396385019|||12|0|2022-10-17
305922|305922.536403.54425|54425|536403|536403|160|Podiatry|Outgoing|Pending|Brown, Matthew|1396385019|||12|0|2022-10-17
305922|305922.536403.54427|54427|536403|536403|83|Other Medical Care|Outgoing|open|Brown,Matthew|1396385019|||12|0|2022-10-17
305922|305922.536015.54428|54428|536015|536015|164|Physiatrist|Outgoing|open|Fortin, Nancy|1679623896|||10|27|2022-10-17
305922|305922.527990.54429|54429|527990|527990|34|Hematology|Outgoing|Pending|Fraser-Farmer, Gail|1841223997|||26|0|2022-10-17
305922|305922.536179.54430|54430|536179|536179|131|Sleep Medicine|Outgoing|open|Stengel, Andrea|1235225871|||27|0|2022-10-17
305922|305922.530128.54416|54416|530128|530128|78|Oral Surgery|Outgoing|open|Winik,Alan|1659584084|Miller,David M||26|0|2022-10-17
305922|305922.536027.54426|54426|536027|536027|78|Oral Surgery|Outgoing|open|Winik,Alan|1659584084|Miller,David M||26|0|2022-10-17
305922|305922.532616.54432|54432|532616|532616|17|Dental General Practice|Outgoing|Pending|DiMattei, Jaclyn|1093238222|||26|0|2022-10-18
305922|305922.532616.54433|54433|532616|532616|77|Optometrist|Outgoing|open|DiMattei, Jaclyn|1093238222|||26|0|2022-10-18
305922|305922.532616.54434|54434|532616|532616|19|Dermatology|Outgoing|open|DiMattei, Jaclyn|1093238222|||26|0|2022-10-18
305922|305922.537759.54435|54435|537759|537759|162|Speech Therapy|Outgoing|Pending|Pomerantz, Janet|1922181510|||25|0|2022-10-18
305922|305922.533101.54437|54437|533101|533101|93|Physical Therapist|Outgoing|Pending|Pomerantz, Janet|1922181510|||25|0|2022-10-18
305922|305922.528403.54438|54438|528403|528403|162|Speech Therapy|Outgoing|open|Pomerantz,Janet|1922181510|||25|0|2022-10-18
305922|305922.533210.54439|54439|533210|533210|162|Speech Therapy|Outgoing|Pending|Pomerantz, Janet|1922181510|||25|0|2022-10-18
305922|305922.535151.54440|54440|535151|535151|72|Occupational Therapy|Outgoing|open|Pomerantz,Janet|1922181510|||25|0|2022-10-18
305922|305922.551993.54441|54441|551993|551993|164|Physiatrist|Outgoing|open|Melnick,Susan|1366565202|||12|0|2022-10-18
305922|305922.552234.54442|54442|552234|552234|72|Occupational Therapy|Outgoing|addressed|DiMattei, Jaclyn|1093238222|||26|0|2022-10-18
305922|305922.536168.54443|54443|536168|536168|162|Speech Therapy|Outgoing|Pending|Pomerantz, Janet|1922181510|||25|0|2022-10-18
