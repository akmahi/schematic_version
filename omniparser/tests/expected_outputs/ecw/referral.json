[
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "533210",
				"id": "533210"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533210.54413",
			"speciality": "Gastroenterology",
			"speciality_code": "28"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "552089",
				"id": "552089"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.552089.54414",
			"speciality": "Optometrist",
			"speciality_code": "77"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "552089",
				"id": "552089"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.552089.54415",
			"speciality": "Urology",
			"speciality_code": "115"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "547456",
				"id": "547456"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.547456.54417",
			"speciality": "Physical Therapist",
			"speciality_code": "93"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "552353",
				"id": "552353"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.552353.54418",
			"speciality": "Speech Therapy",
			"speciality_code": "162"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "550284",
				"id": "550284"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.550284.54419",
			"speciality": "Optometrist",
			"speciality_code": "77"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "533974",
				"id": "533974"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533974.54420",
			"speciality": "Surgery",
			"speciality_code": "135"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "533974",
				"id": "533974"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533974.54421",
			"speciality": "Podiatry",
			"speciality_code": "160"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "524156",
				"id": "14955"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.14955.54422",
			"speciality": "Dermatology",
			"speciality_code": "19"
		}
	},
	{
		"referral": {
			"facility_from": "10",
			"facility_to": "27",
			"patient": {
				"account_no": "536047",
				"id": "536047"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536047.54423",
			"speciality": "Endocrinology",
			"speciality_code": "25"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "536403",
				"id": "536403"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536403.54424",
			"speciality": "Optometrist",
			"speciality_code": "77"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "536403",
				"id": "536403"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536403.54425",
			"speciality": "Podiatry",
			"speciality_code": "160"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "536403",
				"id": "536403"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536403.54427",
			"speciality": "Other Medical Care",
			"speciality_code": "83"
		}
	},
	{
		"referral": {
			"facility_from": "10",
			"facility_to": "27",
			"patient": {
				"account_no": "536015",
				"id": "536015"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536015.54428",
			"speciality": "Physiatrist",
			"speciality_code": "164"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "527990",
				"id": "527990"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.527990.54429",
			"speciality": "Hematology",
			"speciality_code": "34"
		}
	},
	{
		"referral": {
			"facility_from": "27",
			"facility_to": "0",
			"patient": {
				"account_no": "536179",
				"id": "536179"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536179.54430",
			"speciality": "Sleep Medicine",
			"speciality_code": "131"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "530128",
				"id": "530128"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.530128.54416",
			"speciality": "Oral Surgery",
			"speciality_code": "78"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "536027",
				"id": "536027"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536027.54426",
			"speciality": "Oral Surgery",
			"speciality_code": "78"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "532616",
				"id": "532616"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.532616.54432",
			"speciality": "Dental General Practice",
			"speciality_code": "17"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "532616",
				"id": "532616"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.532616.54433",
			"speciality": "Optometrist",
			"speciality_code": "77"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "532616",
				"id": "532616"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.532616.54434",
			"speciality": "Dermatology",
			"speciality_code": "19"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "537759",
				"id": "537759"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.537759.54435",
			"speciality": "Speech Therapy",
			"speciality_code": "162"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "533101",
				"id": "533101"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533101.54437",
			"speciality": "Physical Therapist",
			"speciality_code": "93"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "528403",
				"id": "528403"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.528403.54438",
			"speciality": "Speech Therapy",
			"speciality_code": "162"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "533210",
				"id": "533210"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533210.54439",
			"speciality": "Speech Therapy",
			"speciality_code": "162"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "535151",
				"id": "535151"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.535151.54440",
			"speciality": "Occupational Therapy",
			"speciality_code": "72"
		}
	},
	{
		"referral": {
			"facility_from": "12",
			"facility_to": "0",
			"patient": {
				"account_no": "551993",
				"id": "551993"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.551993.54441",
			"speciality": "Physiatrist",
			"speciality_code": "164"
		}
	},
	{
		"referral": {
			"facility_from": "26",
			"facility_to": "0",
			"patient": {
				"account_no": "552234",
				"id": "552234"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.552234.54442",
			"speciality": "Occupational Therapy",
			"speciality_code": "72"
		}
	},
	{
		"referral": {
			"facility_from": "25",
			"facility_to": "0",
			"patient": {
				"account_no": "536168",
				"id": "536168"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.536168.54443",
			"speciality": "Speech Therapy",
			"speciality_code": "162"
		}
	}
]
